fission-cli/library/Fission/CLI/IPFS/Peers.hs

```diff
@@ -29,4 +28,4 @@ getPeers ::
 getPeers = do
   logDebug @Text "🌌📥 Getting peers remotely"
   Cursor.withHidden $ CLI.Wait.waitFor "Retrieving Fission Peer List..." do
-    ensureM $ sendRequest IPFS.getPeers
+    ensureM $ sendRequest getIPFSPeers
```

Comment:
That's interesting! Why did you decide to switch the naming convention? I personally like the `Something.x` convention, so I'm curious :)

Reply:
I also prefer that convention. This is from the new Servant.Generic client, since it's easier to decompose them in one file rather than importing the outer level into separate files and exposing them there. The tradeoff is that it puts them in a flat namespace.